examples/embeddable_examples/public/react_embeddables/saved_book/saved_book_editor.tsx

```diff
@@ -0,0 +1,208 @@
+    }
+  });
+};
+
+export const SavedBookEditor = ({
+  attributesManager,
+  isCreate,
+  onSubmit,
+  onCancel,
+}: {
+  attributesManager: BookAttributesManager;
+  isCreate: boolean;
+  onSubmit: (addToLibrary: boolean) => void;
+  onCancel: () => void;
+}) => {
+  const [addToLibrary, setAddToLibrary] = React.useState(false);
+  const [authorName, synopsis, bookTitle, numberOfPages] = useBatchedOptionalPublishingSubjects(
+    attributesManager.authorName,
+    attributesManager.bookSynopsis,
+    attributesManager.bookTitle,
+    attributesManager.numberOfPages
+  );
+
+  return (
+    <>
+      <EuiFlyoutHeader hasBorder>
+        <EuiTitle size="m">
+          <h2>
+            {isCreate
+              ? i18n.translate('embeddableExamples.savedBook.editor.newTitle', {
+                  defaultMessage: 'Create new book',
+                })
+              : i18n.translate('embeddableExamples.savedBook.editor.editTitle', {
+                  defaultMessage: 'Edit book',
+                })}
+          </h2>
+        </EuiTitle>
+      </EuiFlyoutHeader>
+      <EuiFlyoutBody>
+        <EuiFormControlLayout>
+          <EuiFormRow
+            label={i18n.translate('embeddableExamples.savedBook.editor.authorLabel', {
+              defaultMessage: 'Author',
+            })}
+          >
+            <EuiFieldText
+              value={authorName}
+              onChange={(e) => attributesManager.authorName.next(e.target.value)}
+            />
+          </EuiFormRow>
+          <EuiFormRow
+            label={i18n.translate('embeddableExamples.savedBook.editor.titleLabel', {
+              defaultMessage: 'Title',
+            })}
+          >
+            <EuiFieldText
+              value={bookTitle}
+              onChange={(e) => attributesManager.bookTitle.next(e.target.value)}
+            />
+          </EuiFormRow>
+          <EuiFormRow
+            label={i18n.translate('embeddableExamples.savedBook.editor.pagesLabel', {
+              defaultMessage: 'Number of pages',
+            })}
+          >
+            <EuiFieldNumber
+              value={numberOfPages}
+              onChange={(e) => attributesManager.numberOfPages.next(+e.target.value)}
+            />
+          </EuiFormRow>
+          <EuiFormRow
+            label={i18n.translate('embeddableExamples.savedBook.editor.synopsisLabel', {
+              defaultMessage: 'Synopsis',
+            })}
+          >
+            <EuiTextArea
+              value={synopsis}
+              onChange={(e) => attributesManager.bookSynopsis.next(e.target.value)}
+            />
+          </EuiFormRow>
+        </EuiFormControlLayout>
+      </EuiFlyoutBody>
+      <EuiFlyoutFooter>
+        <EuiFlexGroup justifyContent="spaceBetween">
+          <EuiFlexItem grow={false}>
+            <EuiButtonEmpty iconType="cross" onClick={onCancel} flush="left">
+              {i18n.translate('embeddableExamples.savedBook.editor.cancel', {
+                defaultMessage: 'Cancel',
```

Comment:
How about changing to "Discard changes"?

Reply:
Resolved in [541b84b](https://github.com/elastic/kibana/pull/182523/commits/541b84bdee44c2d4018f7c12720770bba34681b4)